autonomic-plugin-common/src/main/java/br/com/autonomiccs/autonomic/plugin/common/daos/HostJdbcDao.java

```diff
@@ -99,13 +103,19 @@ public ResourceState getResourceState(long hostId) {
         return hypervisorTypes;
     }
 
-    private String sqlCheckIsThereAnyHostOnCloudDeactivatedByOurManager = "select id from host where removed is null and administration_status = 'ShutDownToConsolidate'";
+    /**
+     * It looks for deactivated hosts in the cloud; if there is at least one host deactivated by the
+     * Autonomiccs platform it returns true.
+     */
     public boolean isThereAnyHostOnCloudDeactivatedByOurManager() {
         List<Long> hostsIds = getJdbcTemplate().queryForList(sqlCheckIsThereAnyHostOnCloudDeactivatedByOurManager, Long.class);
         return CollectionUtils.isNotEmpty(hostsIds);
     }
 
-    private String sqlCheckIsThereAnyHostOnPodDeactivatedByOurManager = "select id from host where removed is null and administration_status = 'ShutDownToConsolidate' and pod_id = ?";
+    /**
+     * It searches for deactivated hosts in the pod with the given id; if find at least one host

```

Comment:
It searches for deactivated hosts in the pod with the given id; if it finds at least one host that was deactivated by the Autonomiccs platform it returns true.